bullets/enemy.py

```diff
@@ -1,5 +1,28 @@
 from .actor import Actor
 
-
 class Enemy(Actor):
-    pass
+    def __init__(maxhealth, patterns, pcb1, pcb2, pcb3 )
+        self.health = maxhealth
+        self.maxhealth = maxhealth
+        self.phase = 0
+        self.status = "Alive"
```

Comment:
i think i like this being `self.is_alive = True` or potentially using a property/getter like [here in player.py](https://github.com/TriCot-Game-Studio/bullets/blob/main/bullets/player.py#L16-L18).